### ISSUE TITLE
Add more Slugs in the list

### ISSUE THREAD

**Issue by vikhyatsingh123** [vikhyatsingh123/SlugTerra#1]
You can see other types of slugs in the website link that I have given in the readme section. From there you can choose any one.
Before contributing, i request you to visit the website once to see the Slugs that are already there. I would love to see more new ones. It's okay if your favourite slug is included and you still wish to contribute the same. But if that's not the case then please add new unique ones.

Thank You!

**Comment by ani-coder-hacker** [comment]
Can I take up this task? I'll add a few more slugs! :)

**Comment by vikhyatsingh123** [comment]
@ani-coder-hacker ya sure, You can contribute in it

**Comment by ankitaugale23** [comment]
Can I work on this?

**Comment by ankitaugale23** [comment]
I added a new slug to the list https://github.com/vikhyatsingh123/SlugTerra/pull/20 here is my PR. Kindly check.

**Comment by gaishhwarya** [comment]
I have added 6 new slugs in the slugs gallery! Kindly Check! (list #24 )

**Comment by mraguso2** [comment]
@vikhyatsingh123 
Fixed merge conflict for yesterdays pull request #10 
should be good now...added new slug Joules

**Comment by satcasm** [comment]
@vikhyatsingh123 in my profile after submitting the PR it is showing ineligible Repo for hacktoberfest, can you please check on that.

**Comment by Rupinderjeet2004** [comment]
yeah please check, after submitting the PR it is showing ineligible Repo for hacktoberfest.

Your PR was submitted to a repository that is not participating in Hacktoberfest. Maintainers of the repository can add the "hacktoberfest" topic to their repository if they wish to participate.

**Comment by varuns1007** [comment]
Can I fix the broken UI and add more slugs?

**Comment by guilhermeorcezi** [comment]
I added a new one slug. Please fix the repository topic to "hacktoberfest", you typo "hactoberfest".

Thank you!

**Comment by vikhyatsingh123** [comment]
OK, I am updating it.

**Comment by jackknight21** [comment]
Hey Vikhyat! I want to contribute on this project

**Comment by vikhyatsingh123** [comment]
@jackknight21 Go ahead!

**Comment by vikhyatsingh123** [comment]
@jackknight21 Happy coding 😊

**Comment by sonalikari** [comment]
Hey @vikhyatsingh123, I want to contribute on this project. Can I ??

**Comment by valak-69** [comment]
Hey @vikhyatsingh123 i want to contribute in this project. Can I ??

**Comment by yogesh-9999** [comment]
Assign this task to me!!!

**Comment by mandiraghosh783** [comment]
hello @vikhyatsingh123 , can i work on this issue and add my favourite card?

**Comment by Krrishn07** [comment]
Hey @vikhyatsingh123 this seems very interesting can I work upon this and provide my contribution,please assign me this.

**Comment by gitamandeep** [comment]
@vikhyatsingh123 please assign this task to me.

**Comment by anshul-132002** [comment]
@vikhyatsingh123 Hey ! i want to contribute in this issue

**Comment by guilhermeorcezi** [comment]
Hello. i want to contribute!

**Comment by VishalAggarwal2** [comment]
Please Give This Task To Me

**Comment by vikhyatsingh123** [comment]
@VishalAggarwal2 @guilhermeorcezi 
 read above comments

**Comment by Chanchal-D** [comment]
I want to contribute

**Comment by vikhyatsingh123** [comment]
**You all can contribute. no need to comment here**

**Comment by SaiKondreddy01** [comment]
Hi @vikhyatsingh123 , I would like to work on this. can you please assign this to me?
